addons/website_sale/static/src/website_builder/products_design_panel.xml

```diff
@@ -452,8 +462,66 @@
 
             <t t-set="imageRatioLabel">Image Ratio</t>
             <t t-set="imageRatioLabelMobile">Mobile Ratio</t>
+
+            <!-- Desktop Ratio -->
+            <t t-set="imgRatios" t-value="[
+                {
+                    label: 'WideScreen (16/9)',
+                    className: 'o_wsale_products_opt_thumb_16_9',
+                },
+                {
+                    label: 'Landscape (4/3)',
+                    className: 'o_wsale_products_opt_thumb_4_3',
+                },
+                {
+                    label: 'Horizontal (6/5)',
+                    className: 'o_wsale_products_opt_thumb_6_5',
+                },
+                {
+                    label: 'Default (1/1)',
+                    className: 'o_wsale_products_opt_thumb_1_1',
+                },
+                {
+                    label: 'Portrait (4/5)',
+                    className: 'o_wsale_products_opt_thumb_4_5',
+                },
+                {
+                    label: 'Vertical (2/3)',
+                    className: 'o_wsale_products_opt_thumb_2_3',
+                },
+            ]"/>
```

Comment:
All these classes are not applied, I think they are overridden by the option on line 545. Maybe we can try to integrate mobile ratios to the existing ratio option ?